commit 8d6838450a3c91e4a1fa8a7bf98fb1d9c39b131e
Author: Lennart Poettering <mznyfn@0pointer.de>
Date:   Sun Aug 31 03:42:56 2008 +0200

    alsa-lib: Don't call the simple mixer interface "legacy"
    
    The simpler mixer interface is not legacy, so the header file shouldn't claim
    otherwise.
    
    Fixes bug#3569
    
    From: Lennart Poettering <mznyfn@0pointer.de>
    Signed-off-by: Jaroslav Kysela <perex@perex.cz>

diff --git a/include/mixer.h b/include/mixer.h
index 59deb9f..df92164 100644
--- a/include/mixer.h
+++ b/include/mixer.h
@@ -88,7 +88,7 @@ typedef int (*snd_mixer_event_t)(snd_mixer_class_t *class_, unsigned int mask,
 
 /** Mixer element type */
 typedef enum _snd_mixer_elem_type {
-	/* Simple (legacy) mixer elements */
+	/* Simple mixer elements */
 	SND_MIXER_ELEM_SIMPLE,
 	SND_MIXER_ELEM_LAST = SND_MIXER_ELEM_SIMPLE
 } snd_mixer_elem_type_t;
@@ -166,7 +166,7 @@ int snd_mixer_class_set_compare(snd_mixer_class_t *class_, snd_mixer_compare_t c
  *  \{
  */
 
-/* Simple (legacy) mixer elements API */
+/* Simple mixer elements API */
 
 /** Mixer simple element channel identifier */
 typedef enum _snd_mixer_selem_channel_id {
